test/integration/transaction-path-test.coffee

```diff
@@ -27,13 +27,12 @@ describe "compiled transaction paths", ->
       expected = "Some API Name:Some Group Name:Some Resource Name:Some Action Name:Example 2"
 
       paths = []
-      dreddTransactions.compile(code, null, (err, {transactions}) ->
+      dreddTransactions.compile(code, null, (err, compilationResult) ->
```

Comment:
I think it would be better to avoid public interfaces with nullable arguments - I am not able to tell what type this method expects in case the argument has a value (without looking into its definition /or documentation).

Reply:
Yeah, the same for me, good point 👍 I had to look up what's the argument after your comment and I figured out it's `filename`. Which has #6 😏